database/influxdb2.go

```diff
@@ -0,0 +1,86 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/gobackup/gobackup/helper"
+	"github.com/gobackup/gobackup/logger"
+)
+
+// InfluxDB v2 database through `influx` cli
+// See https://docs.influxdata.com/influxdb/v2/reference/cli/influx/backup/
+type InfluxDB2 struct {
+	Base
+	host       string
+	token      string
+	bucket     string
+	bucketId   string
+	org        string
+	orgId      string
+	skipVerify bool
+	httpDebug  bool
+}
+
+func (db *InfluxDB2) init() (err error) {
+	viper := db.viper
+	viper.SetDefault("skipVerify", false)
```

Comment:
Use snake_case style for config key. `skip_verify`

Reply:
Done